app/src/main/java/org/p2p/wallet/history/ui/detailsbottomsheet/HistoryTransactionDetailsPresenter.kt

```diff
@@ -28,13 +33,23 @@ class HistoryTransactionDetailsPresenter(
     HistoryTransactionDetailsContract.Presenter {
 
     private val timeFormat = DateTimeFormatter.ofPattern(DateTimeUtils.PATTERN_FULL_DAY, Locale.US)
+    private var tx: HistoryTransaction? = null
+    private var loadUsernameJob: Job? = null
+
+    override fun detach() {
+        super.detach()
+        loadUsernameJob?.cancel()
```

Comment:
won't be it cancelled automatically when the root scope is canceled?

Reply:
ah I always forgout about, you're right